ibis/pandas/core.py

```diff
@@ -117,23 +118,6 @@
 simple_types = scalar_types + (str, type(None))
 
 
-@functools.singledispatch
-def is_computable_input(arg):
```

Comment:
Why are these moved?

Reply:
This function is moved to dispatch.py to avoid circular import. In dispatch.py the default context adjustment function uses this is_computable_input

Reply:
I think it make more sense to keep it in the original place. Not all dispatch methods live in dispatch.py anyway.

Reply:
move_computable_input back to core.py